src/main/groovy/org/transmartproject/core/dataquery/highdim/tworegion/JunctionValues.groovy

```diff
@@ -0,0 +1,16 @@
+package org.transmartproject.core.dataquery.highdim.tworegion
+
+/**
+ * Represents metadata about one junction of a two region variant
+ */
+interface JunctionValues {
+    long getUpEnd();
+    String getUpChromosome();
+    long getUpPos();
+    char getUpStrand();
+    long getDownEnd();
+    String getDownChromosome();
+    long getDownPos();
+    char getDownStrand();
+    boolean getIsInFrame();
```

Comment:
You shouldn't, since that is not the convention (the same way you don't write properties named `getX`; property names should not have verbs). But anyway to get a property named `isInFrame`, you would use `isIsInFrame`.